### ISSUE TITLE
Smarthome integration should expose DisconnectResponse

### ISSUE THREAD

**Issue by proppy** [actions-on-google/actions-on-google-java#45]
Currently it seems that smarthome integration always returns an empty response for DisconnectRequest.

https://github.com/actions-on-google/actions-on-google-java/blob/master/src/main/kotlin/com/google/actions/api/smarthome/SmartHomeApp.kt#L166

This could be problematic as developer might need to return an `errorCode` when the `DisconnectRequest` is failing.

**Comment by Fleker** [comment]
According to the [documentation](https://developers.google.com/assistant/smarthome/develop/process-intents#response_format_4) there is no `errorCode` field for a `DisconnectResponse` type.

**Comment by proppy** [comment]
Yes but https://developers.google.com/assistant/smarthome/develop/process-intents seems to indicate that most response could return `errorCode`

**Comment by Fleker** [comment]
This should be clarified more precisely. Even if the DisconnectResponse provided an error code, I don't believe the platform will interpret it. Regardless it seems like it'll disconnect the service.
